### PR TITLE
I have added the ScreenShot of My Project Idea

### PR DESCRIPTION
**Pull request by praneethaBrindavanam**
Screenshot 2024-11-06 062309
this is under GSSoC'ext 2024.

### CODE REVIEW

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]
open a new pr and add your image 
Also rename your file name 
your name - project title - date

**Review by AKSHITHA-CHILUKA — APPROVED** [review]
rename your file name 
Your name - project title - date

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA  can i write there??

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA can i update my name and project title in this same PR

**Comment by praneethaBrindavanam** [comment]
name: Brindavanam Indira praneetha
project title : Idea-Arca

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA i made the changes kindly check!!

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA kindly label GSSoC'24 
thank you !

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]
please rename your file name

**Comment by praneethaBrindavanam** [comment]
i have already renamed my file name
Idea-Arca.png

File: Submit/Idea-Arca.png
Comment (praneethaBrindavanam) [inline Submit/Idea-Arca.png:1]: Screenshot 2024-11-06 062309

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA  check once

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA  i haved renamed the file !!!!

**Review by AKSHITHA-CHILUKA — CHANGES_REQUESTED** [review]
Your name - Project Title - Date is the format

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA i renamed the file !!!

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA i made the changes !

**Review by AKSHITHA-CHILUKA — APPROVED** [review]

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA thank you !!

**Comment by praneethaBrindavanam** [comment]
@AKSHITHA-CHILUKA  when will the points get reflected!!

**Pull request merged**